plugin/pkg/admission/defaulttolerationseconds/admission.go

```diff
@@ -82,7 +82,19 @@ func (p *Plugin) Admit(attributes admission.Attributes) (err error) {
 
 	toleratesNodeNotReady := false
 	toleratesNodeUnreachable := false
-	for _, toleration := range tolerations {
+	for i, toleration := range tolerations {
+		// replace alpha toleration key with GA key (remove alpha)
+		if toleration.Key == algorithm.DeprecatedTaintNodeNotReady {
+			toleration.Key = algorithm.TaintNodeNotReady
+			pod.Spec.Tolerations[i] = toleration
+		}
+
+		// replace alpha toleration key with GA key (remove alpha)
+		if toleration.Key == algorithm.DeprecatedTaintNodeUnreachable {
+			toleration.Key = algorithm.TaintNodeUnreachable
```

Comment:
same here, this will register as a removal of a DeprecatedTaintNodeUnreachable toleration on an update, which is not allowed